frontend/src/metabase/visualizations/lib/settings/series.js

```diff
@@ -154,19 +155,28 @@ export function seriesSetting({ readDependencies = [], def } = {}) {
     "line.missing": {
       title: t`Replace missing values with`,
       widget: "select",
-      props: {
+      getProps: (_series, settings, _onChange, { settings: vizSettings }) => ({
         options: [
           { name: t`Zero`, value: "zero" },
           { name: t`Nothing`, value: "none" },
-          { name: t`Linear Interpolated`, value: "interpolate" },
+          ...(hasInterpolatedLineMissingOption(
+            settings["display"],
```

Comment:
`display` is a sub setting in `series`, but `stackable._default_stack_type` is a top-level setting.